egov/egov-egi/src/main/java/org/egov/infra/admin/master/service/UserService.java

```diff
@@ -180,14 +190,72 @@ else if (user.getType().equals(UserType.EMPLOYEE))
         createUserRequest.setRequestInfo(microserviceUtils.createRequestInfo());
 
         final RestTemplate restTemplate = new RestTemplate();
+        UserDetailResponse udr = null;
         try {
-            restTemplate.postForObject(createUserServiceUrl, createUserRequest, UserDetailResponse.class);
+            udr = restTemplate.postForObject(createUserServiceUrl, createUserRequest, UserDetailResponse.class);
         } catch (final Exception e) {
```

Comment:
UserService.createUser(user) method is called from PTIS/PGR to create Citizen user. So we need to call M@S api to create user data in microservice schema. Is it okay if we move this entire code which is used for creating user in M@S user service to a MicroserviceUtils and call it from here?